fluent-syntax/src/stream.js

```diff
@@ -24,6 +26,7 @@ export class ParserStream {
     }
 
     this.index++;
+    this.checkLineBreak();
```

Comment:
So, I guess I'd prefer to have an error method "getLineForPos(source, pos)" over this.